coremltools/converters/tensorflow/test/test_tf_2x.py

```diff
@@ -670,6 +670,34 @@ def test_softplus(self):
                                  inputs={input_name: (1, 28, 28)},
                                  outputs=[output_name], decimal=3)
 
+    def test_redundant_transpose(self):
```

Comment:
This is great. 

It would be also good to add a stress test in `MLModelPassesTest` in the file `test_graph_passes.py`, where the test can directly define an mlmodel with different combinations of transposes and check whether the are removed

Reply:
Great idea, I'll add that.

Reply:
PTAL